_rules/links-with-identical-names-serve-equivalent-purpose-b20e66.md

```diff
@@ -152,19 +158,183 @@ A set of two SVG `<a>` elements have the same [accessible name][] and link to th
 </svg>
 ```
 
+#### Passed Example 10
+
+A set of one SVG `a` element and one HTML `a` element that have the same [accessible name][] and link to the [same resource][].
+
+```html
+<a href="http://facebook.com">Follow us"</a>
+
+<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+	<a href="http://facebook.com" aria-label="Follow us">
+		<circle cx="50" cy="40" r="35" />
+	</a>
+</svg>
+```
+
+#### Passed Example 11
+
+All three links have the same [accessible name][]. The second link ("from the light") is only part of the [light tree][]. When the [shadow tree][] is attached to `host` and flattened, this link is overwritten and therefore not part of the [flat tree][] (work in progress). Hence, only the first and third link are considered by this rule and they both point to the [same resource][].
+
+```html
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Links in the shadow</title>
+	</head>
+
+	<body>
```

Comment:
Please only use the content of the body. The doctype, html element, and the head stuff is not relevant to this example. The same goes for most of the new examples below.